_includes/specials/taproot/en/10-ln-with-taproot.md

```diff
@@ -0,0 +1,198 @@
+*By [ZmnSCPxj][]*
+
+In this post, we'll look at two privacy features that [taproot][topic taproot] enables
+for LN:
+
+* [PTLCs][topic ptlc] over LN.
+* P2TR Channels.
+
+### PTLCs Over LN
+
+PTLCs enable [many features][suredbits payment points], with a major
+feature for LN being [payment decorrelation][p4tr ptlcs] without
+any need to randomize routes.[^route-randomization] Every node along a
+single-path or [multipath][topic multipath payments] route can be given a scalar that is used to
+tweak each forwarded PTLC, enabling *payment decorrelation* where
+individual forwards no longer leak the unique identifier for each
```

Comment:
Could link to https://bitcoinops.org/en/newsletters/2021/08/25/#privacy-problems-with-htlcs if you think it works.